tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/signup_info.cpp

```diff
@@ -167,8 +167,10 @@ static tcf_err_t DeserializePublicEnclaveData(
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 std::map<std::string, std::string> CreateEnclaveData(
-    const std::string& originator_public_key_hash
-    ) {
+    uint64_t in_mode,
+    const std::string& in_ext_data,
+    const std::string& in_ext_data_signature,
+    const std::string& in_kme_attestaion) {
```

Comment:
Change kme_attestaion to kme_attestation here and below

Reply:
Done